junit-runtime/src/main/scala/org/scalajs/junit/JUnitTestMetadata.scala

```diff
@@ -0,0 +1,69 @@
+package org.scalajs.junit
+
+import java.lang.annotation.Annotation
+
+import org.junit.FixMethodOrder
+
+import scala.scalajs.js.annotation.JSExportDescendentObjects
+
+@JSExportDescendentObjects
+trait JUnitTestMetadata {
+  def scalajs$junit$metadata(): JUnitClassMetadata
+  def scalajs$junit$newInstance(): ScalaJSJUnitTest
+  def scalajs$junit$invoke(methodId: String): Unit
```

Comment:
Since we always create a synthetic, separate `object` to implement `JUnitTestMetadata`, I don't think there's any reason for these methods to have ugly names. They could just be `metadata`, `newInstance` and `invoke`. They cannot clash with anything, can they?